generator/template_repo.go

```diff
@@ -580,3 +581,12 @@ func dict(values ...interface{}) (map[string]interface{}, error) {
 	}
 	return dict, nil
 }
+
+func escapeTagValue(value string) string {
+	if strings.Contains(value, "`") {
+		panic("Example value contains backtick: " + value)
+	}
+	value = strings.ReplaceAll(value, "\"", "\\\"")
+	value = strings.ReplaceAll(value, "\n", "\\n")
+	return value
```

Comment:
You can use: https://golang.org/pkg/strconv/#Quote

That will make it escape all the necessary chars.

Reply:
Fixed, thanks. I still need to update the tests.